sql/0_0_1-to-1_0_0_upgrade.sql

```diff
@@ -1647,3 +1647,19 @@ UPDATE `clinical_rules` SET `amc_2014_stage1_flag` = 1, `amc_2014_stage2_flag` =
 #IfColumn patient_data cmsportal_login
  DROP COLUMN `cmsportal_login`;
 #EndIf
+
```

Comment:
That's confusing I'm referring to the adds below that comment not the drop statement above it